src/utils/calculations/index.ts

```diff
@@ -0,0 +1 @@
+export * from './calculations';
```

Comment:
You should also put export * from '.calculations' in utils/index.ts file in 1 place. This gives you opportunity to import 
directly from utils

Reply:
Sure thing, forgot about that export